locales/locales.php

```diff
@@ -1662,6 +1680,20 @@ public function __construct() {
 		$pap->wp_locale = 'pap';
 		$pap->slug = 'pap';
 
+		$pap_aw = new GP_Locale();
+		$pap_aw->english_name = 'Papiamento (Aruba)';
+		$pap_aw->native_name = 'Papiamento';
+		$pap_aw->lang_code_iso_639_1 = 'pap';
+		$pap_aw->lang_code_iso_639_2 = 'pap';
```

Comment:
The country code is missing. Can you add `$pap_aw->country_code = 'aw'` please? (https://www.ethnologue.com/country/AW)